README.md

```diff
@@ -1,148 +1,19 @@
-Nakama Swift
-============
+# Nakama Swift
 
-> Swift client for Nakama server.
+**Nakama Swift is deprecated and unsupported**
 
-[Nakama](https://github.com/heroiclabs/nakama) is an open-source server designed to power modern games and apps. Features include user accounts, chat, social, matchmaker, realtime multiplayer, and much [more](https://heroiclabs.com).
+The latest major version of Nakama server is version 3 released January 2021. This Nakama Swift client was developed for Nakama server 1.
```

Comment:
**Nakama Swift is not complete - looking for community contributions.**

The current version of the Nakama Swift SDK is not feature-complete to use with Nakama v2 or Nakama v3 released in January 2021. We are asking for community maintainers to open PR or alternatively send us an [email](mailto:support@heroiclabs.com) if interested in updating and maintaining the SDK on par with other official Nakama Client Libraries.

Reply:
---
What's on Master is actually a partial rewrite of the SDK to use the at-the-time updated Swift + package manager, to be used with Nakama 2+ - but not all SDK functions are stubbed out and most of the realtime functionality is not implemented properly.